README.md

```diff
@@ -34,14 +34,14 @@ Run unit tests with `npm run test`. Jest will pick up test in files with the `.t
 
 To run integration tests:
  
-1. Make sure have chrome and its matching chromedriver installed
+1. Make sure you have Chrome (or Chromium) installed. Depending on the platform you may also need to install chromedriver.
+    - On Ubuntu, `sudo snap install chromium` will install both.
```

Comment:
This worked but I found I still had to add the installed chromedriver to my path for the integration tests to run - it gets installed to `/snap/chromium`. Might be worth mentioning that. 

One of the selenium tests still failed, but it had passed on BuildKite so I'm not going to worry about it!

Reply:
Turns out that was working for me because I had the `chromium-chromedriver` apt package, which installs the snap but also adds extra binaries in the PATH. I've switched the instructions to use that.

Reply:
Also it seems that recent Selenium versions get rid of all that complexity and do it all for us: https://www.selenium.dev/blog/2022/introducing-selenium-manager/